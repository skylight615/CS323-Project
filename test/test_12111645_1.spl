struct Child
{
  float name;
  int age;
};
struct Parent
{
  struct Child c;
  int cc;
};
int test_2_r13()
{
  struct Parent father;
  struct Parent mother;
  father.c.name = 1;
  father.c.age = 1;
  father.cc = 30;
  mother.c.age =father.c.name;
  mother.cc.age = father;
  mother.cc = father;
  return 0;
}