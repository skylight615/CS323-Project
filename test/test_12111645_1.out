Error type 5 at Line 15: unmatching type on both sides of assignment
Error type 5 at Line 18: unmatching type on both sides of assignment
Error type 13 at Line 19: accessing with non-struct variable
Error type 5 at Line 19: unmatching type on both sides of assignment
Error type 5 at Line 20: unmatching type on both sides of assignment
